src/App.jsx

```diff
@@ -1,18 +1,18 @@
 import './App.css';
-import Header from './components/MainPage/Header/Header'
-import Main from './components/MainPage/Main/Main'
-import Footer from './components/MainPage/Footer/Footer'
+import Login from './components/Loginn/Login.js'
+import Regist from './components/Regist/Regist'
+import MainPage from './components/MainPage/MainPage';
+import {Routes , Route , Link} from 'react-router-dom'
 
 
 function App() {
   return (
-    <div>
-      <div className="container">
-      <Header/>
-      <Main/>
-      <Footer/>
-      </div>
-      
+    <div className="App">
+      <Routes>
+        <Route path='/' element={<MainPage/>}/>
+        <Route path='/regist' element={<Regist/>}/>
```

Comment:
route same as the component should be renamed

Reply:
I made changes.